dashboard-frontend/src/types/study.ts

```diff
@@ -0,0 +1,33 @@
+export interface Study {
+  id: string;
+  name: string;
+  slug: string;
+  createdAt: string;
+  progress: StudyProgress;
+  strata: Strata[];
+}
+
+interface StudyProgress {
+  currentParticipants: number;
+  expectedParticipants: number;
+  currentAverageDeviation: number;
+  expectedAverageDeviation: number;
+  date: string;
+  averageDeviationOverTime: DataPoint[];
+}
+
+interface Strata {
+  id: string;
+  percentageDesired: number;
+  currentPercentage: number;
+  remainingPercentage: number;
+  expectedPercentage: number;
+  currentPriceToAdquireParticipant: number;
```

Comment:
"acquire" in english - no "d" ;-

But I actually think it's unnecessarily verbose and not entirely accurate. To some degree, if there were to be multiple prices per participant, "price to acquire" could be the price to get them to start a survey and here we really have the "price per completed survey"...

Options I like better: 

1. `currentParticipantPrice`
2. `currentPricePerParticipant`

Reply:
Yes your right about this, it can be misleading if we have multiple prices.

Both options you proposed look good to me. I will go for **currentPricePerParticipant**,  I think it's slightly more clear than option 1.